### ISSUE TITLE
qmca does not print analyses in same order as datasets are specified

### ISSUE THREAD

**Issue by prckent** [QMCPACK/qmcpack#569]
A minor issue, but at least sometimes the output order is not consistent with the order of datasets specified on the command line:

```
$ qmca -q e -u ev H2O_noj.s000.scalar.dat  H2O.s000.scalar.dat
H2O  series 0  LocalEnergy           =  -462.539950 +/- 0.148661
H2O_noj  series 0  LocalEnergy           =  -460.908017 +/- 0.336092
$ qmca -q e -u ev H2O.s000.scalar.dat  H2O_noj.s000.scalar.dat
H2O  series 0  LocalEnergy           =  -462.539950 +/- 0.148661
H2O_noj  series 0  LocalEnergy           =  -460.908017 +/- 0.336092
```

**Comment by jtkrogel** [comment]
Currently qmca prints out all files alphabetically.  I can change this easily if desired.

**Comment by prckent** [comment]
I prefer the order of specification - here I was going to write "qmca good better best". Order of specification also means the order does not change as new data files are added.

**Comment by jtkrogel** [comment]
@prckent This issue is addressed by PR #600
